src/Home.js

```diff
@@ -111,20 +159,54 @@ const styles = StyleSheet.create({
     color: "white"
   },
   buttonText: {
-    fontSize: 18,
+    fontSize: 16,
     fontWeight: "600",
     padding: 15,
     alignSelf: "center",
     color: "#fff"
   },
-  button: {
-    position: "absolute",
-    bottom: 30,
+  buttonContainer: {
     backgroundColor: "#4696ec",
     width: 150,
     alignSelf: "center",
     borderRadius: 25,
+    shadowOpacity: 0.3,
+    shadowRadius: 2,
+    marginTop: 20,
+    shadowOffset: {
+      height: 1,
+      width: 1
+    }
+  },
+  floatingButton: {
+    position: "absolute",
+    bottom: 30,
+  },
+  inputView: {
+    height: 40,
+    margin: 15,
+    borderWidth: 1,
+    padding: 10,
+  },
+  modalView: {
```

Comment:
Usage of the `View` prefix does not match the naming of the other style keys, suggest using something like `addTodoContainer`. Additionally we may want to audit the style naming as a whole, there is a combination of generic element names, e.g. `buttonContainer` mixed with more specific names like `todoHeading`